### ISSUE TITLE
Make a JOSS publication?

### ISSUE THREAD

**Issue by choldgraf** [executablebooks/meta#150]
What do folks think about making a JOSS publication whenever we release Jupyter Book v0.9? We've had some folks ask us how they can "cite" Jupyter Book, and this could be a way to make it easier for folks to do so. The publication process itself isn't too annoying.

A quick note: I don't think we need to arrive at 1.0 status before doing this, we just need to be explicit in the paper about its current version / status. I think our userbase is going to grow considerably before we get to 1.0 and this issue is mostly about making sure we can take advantage of their interest in citing us.

**Comment by choldgraf** [comment]
Alternatively, we could also submit a more substantial publication elsewhere. For example, when we finished the re-write of Binder we [wrote this SciPy paper](https://conference.scipy.org/proceedings/scipy2018/pdfs/project_jupyter.pdf). It gives you a chance to go into more detail about the goals/principles/design philosophy/etc, but it's also definitely more work

**Comment by jstac** [comment]
I agree and personally I like the idea of JOSS paper.  It would be interesting to go through the review process at JOSS.  Although we couldn't go into as many details, we can always do that in blog posts.  

@mmcky needs to boost his publication output too `:-)`
